apps/wing/src/commands/init.test.ts

```diff
@@ -0,0 +1,88 @@
+import { readdir, writeFile } from "fs/promises";
+import { join } from "path";
+import inquirer from "inquirer";
+import { describe, it, expect, test, vitest, beforeEach, afterEach, vi } from "vitest";
+import { init, initTemplateNames } from "../commands/init";
+import { generateTmpDir } from "src/util";
+
+vitest.mock("inquirer");
+
+describe("initTemplateNames", () => {
+  it("should list init templates", () => {
```

Comment:
I would call these "project templates" instead of "init templates"

Reply:
Updated